mods/tuxemon/maps/spyder_bedroom.tmx

```diff
@@ -67,6 +67,7 @@
     <property name="act1" value="translated_dialog spyder_papertown_restinbed"/>
     <property name="act2" value="set_monster_health ,"/>
     <property name="act3" value="set_monster_status ,"/>
+    <property name="act4" value="set_variable lastcenter:spyder_healing_center.tmx"/>
```

Comment:
This is a different map?

Reply:
Exactly, if you heal in the bedroom, then it sets the variable in Cotton Town (Paper Town > Route1 > Cotton Town). So if you lose against a wild monster in Route 1, then you'll be teleported to the Cotton Town center. I'm not convinced.

Reply:
Why not teleport back here?

Reply:
Yeah, I'm looking into. It's because of this: 
`teleport = [player.game_variables["lastcenter"], 7, 10]`
7,10 doesn't exist in the bedroom, the bedroom is 11x9.
Now I'm checking, I'll update as soon as I fix it.